examples/object_detection_2d/README.md

```diff
@@ -47,3 +49,32 @@ optional arguments:
   --test-suite TEST_SUITE
                         Optionally specify a test suite to test. Test against all available test suites when unspecified.
 ```
+
+### Pre-built Workflow Extension
+
+A pre-built workflow can be extended simply by re-defining the [`workflow`](object_detection_2d/extended/workflow.py). In
+this example, we added a new boolean field `occluded` to each `GroundTruth` `LabeledBoundingBox` by extending it.
+
+```python
+@dataclass(frozen=True)
+class ExtendedBoundingBox(LabeledBoundingBox):
+    occluded: bool
+
+@dataclass(frozen=True)
+class GroundTruth(BaseGroundTruth):
+    bboxes: List[ExtendedBoundingBox]
+```
+
+And define a new workflow with the extended `GroundTruth` class:
+
+```python
+_workflow, TestCase, TestSuite, Model = define_workflow(
```

Comment:
Nit: I'd omit the `workflow` in favor of just `_`. At some point (1.0?) we will likely remove workflow from this signature as it isn't particularly useful.